packages/oui-pagination/README.md

```diff
@@ -0,0 +1,125 @@
+# oui-pagination
+
+<component-status cx-design="partial" ux="prototype"></component-status>
+
+oui-pagination is a package which provide styles for the pagination component.
+
+## Installation
+
+```less
+@import 'oui-pagination/pagination';
+```
+
+## Usage
+
+### Button group
+
+```html:preview
+<div class="oui-pagination">
+  <oui-dropdown arrow>
+    <a tabindex="-1"
```

Comment:
You should use a button instead of an `a` since you're not redirect or changing the client route. Also, you could use `oui-button_link` to have a 40px height link button, so it could be better on mobile.